composed_api.rst

```diff
@@ -218,29 +209,25 @@ Example
 
 ::
 
-    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v1/aoi/123?source=grid
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/aoi/123/?source=grid
 
 ::
 
     {
-        "GRiD API": {
-            "API Version": "v1"
+        "API Version": "v2", 
```

Comment:
It might be lint to confirm that the code handling the request is the correct API version.